packages/crowdstrike/data_stream/fdr/elasticsearch/ingest_pipeline/default.yml

```
@@ -1970,56 +1963,28 @@ processors:
         - append:
             field: error.message
             value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
-  - set:
-      field: source.ip
-      if: ctx.source?.ip == null && ctx.crowdstrike?.LocalAddressIP6 instanceof List && ctx.crowdstrike.LocalAddressIP6.length > 0
-      value: "{{{crowdstrike.LocalAddressIP6.0}}}"
-  # - foreach:
-  #     if: ctx.crowdstrike?.LocalAddressIP6 != null && ctx.crowdstrike.LocalAddressIP6 instanceof List && ctx.crowdstrike.LocalAddressIP6.length > 0
-  #     field: crowdstrike.LocalAddressIP6
-  #     processor:
-  #       append:
-  #         field: source.ip
-  #         value: '{{{_ingest._value}}}'
-  #         allow_duplicates: false
-  - set:
-      field: source.address
-      copy_from: source.ip
-      ignore_empty_value: true
-  - rename:
-      field: crowdstrike.LocalPort
-      target_field: source.port
-      ignore_missing: true
   - convert:
       tag: convert_RemoteAddressIP4_ip
       field: crowdstrike.RemoteAddressIP4
       type: ip
       ignore_missing: true
-  - rename:
-      field: crowdstrike.RemoteAddressIP4
-      target_field: destination.ip
-      ignore_missing: true
-  - set:
-      field: destination.address
-      copy_from: destination.ip
-      ignore_empty_value: true
   - convert:
       tag: convert_RemoteAddressIP6_ip
       field: crowdstrike.RemoteAddressIP6
       type: ip
       ignore_missing: true
-  - rename:
-      field: crowdstrike.RemoteAddressIP6
-      target_field: destination.ip
-      ignore_missing: true
-  - set:
-      field: destination.address
-      copy_from: destination.ip
-      ignore_empty_value: true
-  - rename:
-      field: crowdstrike.RemotePort
-      target_field: destination.port
-      ignore_missing: true
+
+  - pipeline:
+      tag: pipeline_outbound_network
+      if: ctx.network?.direction != 'inbound'
+      name: '{{ IngestPipeline "outbound_network" }}'
```

```suggestion
      name: '{{ IngestPipeline "not_inbound_network" }}'
```
WDYT? because it corresponds to both outbound and unknown?

---

I would prefer not; I'm embedding some cognitive dissonance here to ensure that we don't get comfortable with what we are doing. I'll add a comment.